pkg/workloads/tf_api/requirements.txt

```diff
@@ -2,15 +2,3 @@ flask==1.0.2
 Flask-API==1.1
```

Comment:
Do you think we should be explicit and call this file "requirements.cortex.txt" or something similar?

Reply:
will do